docs/modules/reference/pages/glossary.adoc

```diff
@@ -1,218 +1,247 @@
 
-Message broker:: When using Minions and Sentinels, a message broker is required for communication between servers.
+message broker:: When using Minions and Sentinels, a message broker is required for communication between servers.
 {page-component-title} has an embedded ActiveMQ server available for smaller deployments.
-You can swap the broker with a dedicated ActiveMQ, gRPC, or Kafaka cluster to provide scalability and load balancing.
+You can swap the broker with a dedicated ActiveMQ, gRPC, or Kafka cluster to provide scalability and load balancing.
+
+Minion:: A distributed component that enables {page-component-title} to monitor devices and services in locations that it normally cannot reach.
+Minions communicate with these remote devices while {page-component-title} performs coordination and task delegation (see xref:development:minion/introduction.adoc[]).
+
+monitoring location:: Also referred to as "Minion location" or "location."
+Nodes assigned to a location are monitored by Minions assigned to the same location. +
+This is distinct from geolocation, which is the actual physical location of a monitored entity.
+
+== N
 
-xref:development:minion/introduction.adoc#minion[Minion]:: A distributed component that enables OpenNMS to monitor devices and services in locations that OpenNMS cannot reach.
-Minions communicate with these remote devices while OpenNMS performs coordination and task delegation.
+https://github.com/OpenNMS/nephron[Nephron]:: An OpenNMS component that enables horizontal scaling of flow processing.
+It is not required for flow processing, but can improve performance for deployments with a very high volume of flow documents.
 
-Nephron:: An OpenNMS component that enables horizontal scaling of flow processing.
-Not required for flow processing, but can improve performance for deployments with a very high volume of flow documents.
+network operations center (NOC):: One or more locations where a network is monitored and controlled (see https://en.wikipedia.org/wiki/Network_operations_center[Network Operations Center]).
 
-Network Operations Center (NOC):: One or more locations where a network is monitored and controlled (see https://en.wikipedia.org/wiki/Network_operations_center[Network Operations Center]).
+xref:deployment:time-series-storage/newts/introduction.adoc[New-Fangled Time Series Datastore (Newts)]:: An OpenNMS time series datastore based on Apache Cassandra.
+You can use it as an alternative persistence strategy (instead of JRobin or RRDtool).
 
-xref:deployment:time-series-storage/newts/introduction.adoc#ga-opennms-operation-newts[New-Fangled Time Series Datastore (Newts)]:: An OpenNMS time series datastore based on Apache Cassandra.
-Use it as an alternative persistence strategy instead of JRobin or RRDtool.
+node:: A device or application that sends communications over a network (for example, a server).
+They can be provisioned to and monitored by {page-component-title}.
```

Comment:
```suggestion
They can be provisioned to and monitored by {page-component-title}.
A node must include at least one IP interface, otherwise it is removed from your {page-component-title} inventory.
```